sample_cgfs/rv32ip.cgf

```diff
@@ -0,0 +1,4419 @@
+    opcode:
+      kdmbb: 0
+    rs1:
+      <<: *all_regs
+    rs2:
+      <<: *all_regs
+    rd:
+      <<: *all_regs
+    op_comb:
+      <<: *rfmt_op_comb
+    val_comb:
+      abstract_comb:
+        'simd_base_val("rs1", xlen, 16, signed=True)': 0
+        'simd_base_val("rs2", xlen, 16, signed=True)': 0
+        'simd_val_comb(xlen, 16, signed=True)': 0
+
+kdmbt:
+    config:
+      - check ISA:=regex(.*I.*P.*)
+    opcode:
+      kdmbt: 0
+    rs1:
+      <<: *all_regs
+    rs2:
+      <<: *all_regs
+    rd:
+      <<: *all_regs
+    op_comb:
+      <<: *rfmt_op_comb
+    val_comb:
+      abstract_comb:
+        'simd_base_val("rs1", xlen, 16, signed=True)': 0
+        'simd_base_val("rs2", xlen, 16, signed=True)': 0
+        'simd_val_comb(xlen, 16, signed=True)': 0
+
+kdmtt:
+    config:
+      - check ISA:=regex(.*I.*P.*)
+    opcode:
+      kdmtt: 0
+    rs1:
+      <<: *all_regs
+    rs2:
+      <<: *all_regs
+    rd:
+      <<: *all_regs
+    op_comb:
+      <<: *rfmt_op_comb
+    val_comb:
+      abstract_comb:
+        'simd_base_val("rs1", xlen, 16, signed=True)': 0
+        'simd_base_val("rs2", xlen, 16, signed=True)': 0
+        'simd_val_comb(xlen, 16, signed=True)': 0
+
+kslraw:
+    config:
+      - check ISA:=regex(.*I.*P.*)
+    opcode:
+      kslraw: 0
+    rs1:
+      <<: *all_regs
+    rs2:
+      <<: *all_regs
+    rd:
+      <<: *all_regs
+    op_comb:
+      <<: *rfmt_op_comb
+    val_comb:
+      abstract_comb:
+        'simd_base_val("rs1", xlen, 32, signed=True)': 0
+        'simd_base_val("rs2", xlen, 32, signed=True)': 0
+        'simd_val_comb(xlen, 32, signed=True)': 0
+
+kslraw.u:
+    config:
+      - check ISA:=regex(.*I.*P.*)
+    opcode:
+      kslraw.u: 0
+    rs1:
+      <<: *all_regs
+    rs2:
+      <<: *all_regs
+    rd:
+      <<: *all_regs
+    op_comb:
+      <<: *rfmt_op_comb
+    val_comb:
+      abstract_comb:
+        'simd_base_val("rs1", xlen, 32, signed=False)': 0
+        'simd_base_val("rs2", xlen, 32, signed=True)': 0
+        'rs1_val == rs2_val': 0
+        'rs1_val != rs2_val': 0
+
+ksllw:
+    config:
+      - check ISA:=regex(.*I.*P.*)
+    opcode:
+      ksllw: 0
+    rs1:
+      <<: *all_regs
+    rs2:
+      <<: *all_regs
+    rd:
+      <<: *all_regs
+    op_comb:
+      <<: *rfmt_op_comb
+    val_comb:
+      abstract_comb:
+        'simd_base_val("rs1", xlen, 32, signed=True)': 0
+        'walking_ones("rs2_val", ceil(log(32, 2)), False)': 0
+        'walking_zeros("rs2_val", ceil(log(32, 2)), False)': 0
+        'alternate("rs2_val", ceil(log(32, 2)), False)': 0
+
+kslliw:
+    config:
+      - check ISA:=regex(.*I.*P.*)
+    opcode:
+      kslliw: 0
+    rs1:
+      <<: *all_regs
+    rd:
+      <<: *all_regs
+    op_comb:
+      <<: *ifmt_op_comb
+    val_comb:
+      abstract_comb:
+        'simd_base_val("rs1", xlen, 32, signed=True)': 0
+        'simd_imm_val("imm_val", 5)': 0
+
+kdmabb:
+    config:
+      - check ISA:=regex(.*I.*P.*)
+    opcode:
+      kdmbb: 0
```

Comment:
Possible wrong opcode. Please recheck.

Reply:
Fixed in b30bb541f6e6536d4c37e8f604a59ac5215514c8